azavea/green-equity-demo
# Lazy load high res states

## Overview

This PR adds low and high resolution state boundary geojson files and code to dynamically load them in the UI (low-res first). This was intended to solve the issue of their being small gaps between the states, but the higher resolution data set seems only to have made these gaps more apparent.

I'm still opening a PR because the the dynamic high-res load could improve page load times.

Closes #72 

### Notes

I used the bbox-remove script because QGIS wouldn't export the shapes without them (even if explicitly set) and typescript would complain because the infered type was `number[]` instead of `[number, number, number, ...]`.

## Testing Instructions

- Open network tab
- http://localhost:8765/
- Check for separate request for high res data

 ## Checklist
